manim/__main__.py

```diff
@@ -150,23 +151,38 @@ def get_module(file_name):
 
 
 def main():
-    module = get_module(file_writer_config["input_file"])
-    all_scene_classes = get_scene_classes_from_module(module)
-    scene_classes_to_render = get_scenes_to_render(all_scene_classes)
-    sound_on = file_writer_config["sound"]
-    for SceneClass in scene_classes_to_render:
-        try:
-            # By invoking, this renders the full scene
-            scene = SceneClass()
-            open_file_if_needed(scene.file_writer)
-            if sound_on:
-                play_finish_sound()
-        except Exception:
-            print("\n\n")
-            traceback.print_exc()
-            print("\n\n")
-            if sound_on:
-                play_error_sound()
+    if hasattr(args,"subcommands"):
+        if "cfg" in args.subcommands:
+            if args.cfg_subcommand is not None:
+                subcommand=args.cfg_subcommand
+                if subcommand == "write":
+                    cfg_file_utils.write(args.level)
+                elif subcommand == "show":
+                    cfg_file_utils.show()
+                elif subcommand == "export":
+                    cfg_file_utils.export(args.dir)
+            else:
+                logger.error("NO ARGUMENT PROVIDED, exiting...")
```

Comment:
Nitpick: no need to yell ;)

Reply:
I'll change that lol. Just thought it would get the user's attention.